saleor/graphql/core/mutations.py

```diff
@@ -495,10 +502,13 @@ def get_instance(cls, info, **data):
         object_id = data.get("id")
         qs = data.get("qs")
         if object_id:
-            model_type = cls.get_type_for_model()
-            instance = cls.get_node_or_error(
-                info, object_id, only_type=model_type, qs=qs
-            )
+            if cls._meta.model._meta.object_name == "ShippingMethod":
```

Comment:
I am not convinced that specific `ShippingMethod` cases should be placed in base `Core` mutation. IMHO, get_instance should be overwrite for any non-standard cases (like ShippingMethod) but inside a mutation dedicated to this non-standard case, not inside a Base mutation.